.gitignore

```diff
@@ -16,3 +16,4 @@ docs/graphql-api/public
 *.zip
 .VSCodeCounter
 .DS_Store
+.DS_Store
```

Comment:
now we have it twice :laughing:

Reply:
let's make it three then! 
weird so, let me check...